setup.cfg

```diff
@@ -1,5 +1,9 @@
 [pytest]
-addopts = -v --cov zdict --cov-report html --cov-report term
+flake8-ignore =
+    zdict/__init__.py ALL
+    zdict/dictionaries/template.py ALL
+
+addopts = --flake8 -v --cov zdict --cov-report html --cov-report term
```

Comment:
ok. so if I want the flake8 syntax checking in local env, I should use `py.test --flake8` instead of `py.test`.
I'm gonna remove the `--flake8` in `setup.cfg`.